### DISCUSSION TITLE
Layout is Changed

### DISCUSSION THREAD

**Discussion by haranobuhardo** [importantimport/urara#85] [Q&A]
I don't know why, but I believe last night my blog (based on Urara) was looking good, with the profile card placed on the left, and the blog post list located on the right. But today, it just looks like one column, with all the profile and post stacked vertically.
![image](https://github.com/user-attachments/assets/3f508d20-feb4-4933-9c15-eac0929b13a7)

I also checked Urara's demo site, and it also looks the same
![image](https://github.com/user-attachments/assets/41663d4e-c1e7-4a77-bd05-add0a447045c)

Did I miss something? Is my forked Urara dependent to Urara's demo site?

**Comment by kwaa** [comment]
Urara displays different layouts depending on your screen size, probably because your screen is not wide enough or the browser is not maximized.

**Reply by haranobuhardo** [reply]
It's weird, I need to zoom-out to 90% to see the responsive back to desktop view (tried with 2-laptop). Even my tablet (which can't zoomout page), also doesn't show desktop view layout.

**Reply by kwaa** [reply]
Urara uses Tailwind CSS's `xl` breakpoint, which is 1280px.

ref: https://tailwindcss.com/docs/theme#screens

**Reply by haranobuhardo** [reply]
Thanks for clarifying this! I thought you were using md.  Appreciate your help.
